dev/statement_serializer.h

```diff
@@ -1788,10 +1782,14 @@ namespace sqlite_orm {
                         ss << "(";
                     }
                     ss << "SELECT ";
+                    call_if_constexpr<is_rowset_deduplicator_v<T>>(
```

Comment:
nit: probably it would be more readable if we replace `T` here with `statement_type::return_type` (e.g. we can declare `using return_type = typename statement_type::return_type` just below `using statement_type`